scripts/pybullet_backend.py

```diff
@@ -151,8 +151,8 @@ def main():
         # images to save time.
         import trifinger_object_tracking.py_tricamera_types as tricamera
 
-        # spawn a cube in the arena
-        cube = collision_objects.Block()
+        # spawn a cube in the centre of the arena
+        cube = collision_objects.Block(position=[0.0, 0.0, 0.035])
```

Comment:
are we purposefully starting a bit above the stage for the block? I was just checking the block half-size is 0.0325..

Reply:
Yes, the default in `Block` is actually even higher, so I already lowered it.  Not sure if this would really be an issue but I think it is good to spawn it a little bit higher as needed to make sure there are no collisions with the ground.